source/index.html.erb

```diff
@@ -11,12 +11,13 @@ created_by:
   email: james@lazyatom.com
   name: James Adam
 layout: base
+description: The London Ruby User Group (LRUG) meets on the 2nd Monday of the month for talking and socialising (about Ruby).
```

Comment:
![image](https://user-images.githubusercontent.com/325384/93710049-51aa9f00-fb3b-11ea-8519-eb925a79a215.png)

I wanted the homepage to have a description which uses all our keywords over being blank, this worked nicely.